### PR TITLE
feat: Blindex integration

### PR DESCRIPTION
**Pull request by MichalSzreder**

### CODE REVIEW

**Review by tima-t — COMMENTED** [review]
LGTM

File: test/e2e/import_wallet.spec.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ describe('Import wallet-["MAINNET"]', async () => {
     ).equals(ethAddress)
 
     // Check RSK & ERC20 tokens
-    const rskTokens = ['RBTC', 'SOV', 'FISH']
+    const rskTokens = ['RBTC', 'SOV', 'FISH', 'BDX', 'BDUS', 'BDEU', 'XUSD', 'ETHs', 'bXAU', 'bGBP']
```
Comment (tkeidar) [inline test/e2e/import_wallet.spec.js:173]: Add DOC

File: test/pages/OverviewPage.js [hunk 2]
```diff
@@ -213,6 +213,13 @@ class OverviewPage {
 
       case 'SOV':
       case 'FISH':
+      case 'BDX':
```
Comment (tkeidar) [inline test/pages/OverviewPage.js:216]: Add DOC

**Review by tkeidar — CHANGES_REQUESTED** [review]
add DOC on two spots

File: test/e2e/import_wallet.spec.js (hunk 1 above)
Comment (MichalSzreder) [reply test/e2e/import_wallet.spec.js]: done

File: test/pages/OverviewPage.js (hunk 2 above)
Comment (MichalSzreder) [reply test/pages/OverviewPage.js:216]: done

**Review by tima-t — COMMENTED** [review]
If possible, the icons could be in circles and follow our standard. 

We already have an icon for ETH so you can reuse it 

Beside that LGTM

**Comment by MichalSzreder** [comment]
> If possible, the icons could be in circles and follow our standard.
> 
> We already have an icon for ETH so you can reuse it
> 
> Beside that LGTM

I have updated icons

**Pull request closed without merging**